### PR TITLE
Add comments & commentsCount

### PR DESCRIPTION
**Pull request by MrNice42**

### CODE REVIEW

**Comment by seeden** [comment]
Thanks

File: src/Comments.jsx [hunk 1]
```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import FacebookProvider, {Like, Share} from 'react-facebook';
```
Comment (efosao) [inline src/Comments.jsx]: @MrNice42 To fix broken build, use relative imports instead importing from 'react-facebook'.

```js
import FacebookProvider from './FacebookProvider';
import Like from './Like';
import Share from './Share';
```

File: src/CommentsCount.jsx [hunk 2]
```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import FacebookProvider, {Like, Share} from 'react-facebook';
```
Comment (efosao) [inline src/CommentsCount.jsx]: @MrNice42 Same here... ( To fix broken build, use relative imports instead importing from 'react-facebook' ).

```js
import FacebookProvider from './FacebookProvider';
import Like from './Like';
import Share from './Share';
```

**Comment by MrNice42** [comment]
ah; my build system was making it working ^^'
should be ok now :)

**Pull request merged**